docs/developer/quickstart.md

```diff
@@ -121,6 +121,7 @@ This will build the docker images `nginx-gateway-fabric:<your-user>` and `nginx-
 
 > Note: You will need a valid NGINX Plus license certificate and key named `nginx-repo.crt` and `nginx-repo.key` in the
 > root of this repo to build the NGINX Plus image.
+> You will also need a valid NGINX Plus JSON Web Token (JWT) to deploy NGF with NGINX Plus. That JWT should be stored in the `license.jwt` file in the root of the `nginx-gateway-fabric/` directory.
```

Comment:
We can probably link a section on a documentation page that explains how to download one.

Likely unnecessary in context, but would lead a reader to Plus stuff if they happened to go straight into contributing processes for some reason.